### PR TITLE
codeql for Canvas-LMS-Release-2023-03-15.23

### PR DESCRIPTION
**Pull request by Aditi8007**

### CODE REVIEW

**Review by sudhirverma — COMMENTED** [review]
Please add criticality_score too.

File: canvas-lms/release-2023-03-15.23/sast/canvas-lms-release-2023-03-15.23-codeql-report.json [hunk 1]
```diff
@@ -0,0 +1,239 @@
+{\rtf1\ansi\ansicpg1252\cocoartf2639
```
Comment (sudhirverma) [inline canvas-lms/release-2023-03-15.23/sast/canvas-lms-release-2023-03-15.23-codeql-report.json:1]: @Aditi8007 this is not a proper json format

File: pimcore/10.1.0/scorecard/pimcore-v10.1.0-scorecard-report.json [hunk 2]
```diff
@@ -0,0 +1,236 @@
+{
```
Comment (sudhirverma) [inline pimcore/10.1.0/scorecard/pimcore-v10.1.0-scorecard-report.json:1]: can you fix the version detail you have created 10.1.0 version folder and in file you have use v10.1.0

**Review by sudhirverma — CHANGES_REQUESTED** [review]

**Review by asa1997 — COMMENTED** [review]
@Aditi8007 @harimohanr Duplicate pr. Hence closing.

**Pull request closed without merging**